tests/fgcmcalTestBase.py

```diff
@@ -92,16 +88,18 @@ def _runPipeline(self, repo, pipelineFile, queryString=None,
             Pipeline definition file.
         queryString : `str`, optional
             String to use for "-d" data query.
-        inputCollections : `str`, optional
-            String to use for "-i" input collections (comma delimited).
+        inputCollections : `list` [`str`], optional
+            Input collections list.
         outputCollection : `str`, optional
-            String to use for "-o" output collection.
+            Output collection name.
         configFiles : `list` [`str`], optional
-            List of config files to use (with "-C").
+            List of config files to use.  Each string will be of the form
```

Comment:
I'd maybe replace "to use" with "to load, in order".

Similarly for the configOptions below.